static/sass/_pattern_cards.scss

```diff
@@ -13,6 +13,22 @@
       color: $color-x-light;
       padding: calc(#{$spv-inner--large} - 1px);
     }
+
+    &.is-highlighted {
```

Comment:
perhaps you should call this class something else (has-extra-padding) or something like this, as the card is already highlighted?

Reply:
I'm trying to stick to the convention where highlighted cards are the ones with a shadow.

Reply:
[vanilla highlighted card](https://vanillaframework.io/docs/patterns/card#highlighted)